src/main/java/com/group24/decide/Decide.java

```diff
@@ -90,32 +103,21 @@ private boolean checkPUM(boolean[][] PUM, int index) {
 
     /**
      *
-     * @param args
+     * @return True if the entire vector FUV contains true values. Else false.
      */
-    public static void main(String[] args){
-
-        Parameter parameters = new Parameter(1,0,0,0,0,0,0,0,0,0,0,0,0,0,
-                0,0,0,0,0);
 
-        Datapoints[] Points = new Datapoints[NUMPOINTS];
-        Points[0] = new Datapoints(1, 2); // Input the data points
-        Points[1] = new Datapoints(1,2);
-        lic = new LIC(parameters, Points);
-
-        // Input all values for LCM.
-        CONNECTORS[][] LCM = {
-                {CONNECTORS.ANDD, CONNECTORS.ANDD, CONNECTORS.ORR, CONNECTORS.ANDD, CONNECTORS.NOTUSED},
-                {CONNECTORS.ANDD, CONNECTORS.ANDD, CONNECTORS.ORR, CONNECTORS.ANDD, CONNECTORS.NOTUSED}
-        };
-
-        CMV = lic.runLICConditions(CMV.length); // Run the LIC conditions
-
-        LAUNCH = true;
-        for(int i=0;i<FUV.length;i++){
-            if( !FUV[i] ){
-                LAUNCH = false;
-                break;
+    public boolean launch() {
+        calcCMV();
+        calcPUM();
+        calcFUV();
+        if (FUV.length == 0) {
+            return false;
+        }
+        for (boolean b : FUV) {
+            if (!b) {
+                return false;
             }
         }
+        return true;
```

Comment:
The function which determines this boolean signal should be called DECIDE() according to the specification.